qiskit/optimization/converters/integer_to_binary_converter.py

```diff
@@ -195,6 +196,10 @@ def _substitute_int_var(self):
         self._dst.linear_constraints.add(lin_expr, linear_sense, linear_rhs, linear_ranges,
                                          linear_names)
 
+        # TODO: add quadratic constraints
+        if self._src.quadratic_constraints.get_num() > 0:
+            warnings.warn("Quadratic constraints are currently not suppourted yet.")
```

Comment:
Should this only warn here or raise an error? What do you think makes sense from user-perspective?